### DISCUSSION TITLE
Would love to be a part of this project.

### DISCUSSION THREAD

**Discussion by Yashv2211** [hsbc/pyratings#6] [Q&A]
Hey Andreas,

I am a data science student and I am keen on working on financial metrics and data. I would love to contribute to this project or any project (that you may find my skills fit for) if needed in any way. 

I would also attach my cv so you can look at my skillset.

Please do reply if you find time, some wise words are always appreciated.

Kind Regards
Yashvardhan Singh
[Yashvardhan_Singh_CV.docx](https://github.com/hsbc/pyratings/files/9486975/Yashvardhan_Singh_CV.docx)

**Comment by a-vester** [comment]
Hi Yashvardhan,

Thank you very much for your interest in ``pyratings``.

Feel free to contribute to this project.  If you can think about any kind of improvement or if you find a bug, don't hesitate to open an [issue](https://github.com/hsbc/pyratings/issues/new) where we can then discuss things further.

Contributions will be appreciated.

Cheers,
Andreas
